src/app/(private)/challenge/[id]/progress/page.tsx

```
@@ -21,55 +34,125 @@ const Progress = () => {
     bottomBackgroundColor: '#FFF',
   });
 
+  const { data: data, isLoading } = useGetChallengeProgressLog(id as string);
```

Biome (1.9.4)

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)